dm/worker/worker.go

```diff
@@ -214,7 +214,7 @@ func (w *Worker) StartSubTask(cfg *config.SubTaskConfig) {
 	defer w.Unlock()
 
 	// copy some config item from dm-worker's config
-	w.copyConfigFromWorker(cfg)
+	copyConfigFromSource(cfg, w.cfg)
```

Comment:
Does this still need?

Reply:
Others such as `worker server` also call `StartSubTask` and use origin subtaskCfgs